docs/writing-docs/macros.md

```diff
@@ -206,9 +206,15 @@ To avoid changing screenshots, PXT automatically renders code snippets to blocks
 You need declare the packages required to load your snippet, unless they are part of the default empty template.
```

Comment:
"Extensions" formerly called "packages"?

Reply:
Good catch, maybe we should add ```extensions as an alias for the packages macro too so we can phase that out over time too

Reply:
file a bug :)